.github/workflows/ci.yml

```diff
@@ -138,10 +137,13 @@ jobs:
   build_macos:
     strategy:
       matrix:
-        platform: [macos-latest, macos-10.15]
+        platform: [macOS-latest, macOS-11.6, macOS-10.15]
```

Comment:
It would be great that you specify that we want execution on ARM machine specifically (https://github.community/t/can-i-use-a-mix-of-self-hosted-github-hosted-runners-in-a-single-matrix-job/117286/2), something like this, for at least of of the runs

Reply:
The version manages the pick up the ARM machine since Github does not offer it, but what we really want is CI to run on ARM macOS machine and that would be more specific